### PR TITLE
RHINENG-8764: update "is_ros" field of system_profile

### PR DESCRIPTION
**Pull request by JoySnow**
- add "is_pcp_raw_data_collected" field to system_profile
 - bump insights-core from 3.3.1 to 3.3.18

### CODE REVIEW

**Comment by JoySnow** [comment]
Hi @xiangce, @upadhyeammit, please kindly help review the changes. & feel free to let me know if any concerns/thoughts.

File: src/puptoo/process/profile.py [hunk 1]
```diff
@@ -624,8 +630,10 @@ def system_profile(
             profile["releasever"] = profile["yum_updates"].get("releasever")
             profile["basearch"] = profile["yum_updates"].get("basearch")
 
-    if pmlog_summary or ros_config:
+    if (pmlog_summary_pcp_zeroconf or ros_collect or    # ros new collection
+            pmlog_summary or ros_config):               # ros old collection
         profile["is_ros"] = True
+        profile["is_pcp_raw_data_collected"] = bool(pcp_raw_data)
```
Comment (r14chandra) [inline src/puptoo/process/profile.py:636]: Using this field, you need to add it under `platform_metadata` as well [here](https://github.com/RedHatInsights/insights-puptoo/blob/1469fc638441724f3444971f7616f626ea388330/src/puptoo/app.py#L246-L247)

**Review by r14chandra — CHANGES_REQUESTED** [review]

File: src/puptoo/process/profile.py (hunk 1 above)
Comment (JoySnow) [reply src/puptoo/process/profile.py:636]: @r14chandra, thanks for the review.    

Updated the code to [send "is_pcp_raw_data_collected" under platform_metadata](https://github.com/RedHatInsights/insights-puptoo/pull/329/commits/41707f2d164fcbd0b2be373d1573372b3111bb57)

**Review by r14chandra — APPROVED** [review]
Approving the PR. But I'll still wait for @upadhyeammit for the logic he wants.

**Review by upadhyeammit — APPROVED** [review]

File: src/puptoo/process/profile.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
 from insights.parsers.systemctl_status_all import SystemctlStatusAll
 from insights.parsers.eap_json_reports import EAPJSONReports
 from insights.specs import Specs
-
+from insights.specs.datasources.pcp import ros_collect as ds_ros_collect
```
Comment (xiangce) [inline src/puptoo/process/profile.py:51]: @JoySnow -  Generally,`datasource` is for core collection, it's not a good idea to use it directly in plugins like puptoo. Please use the `insights.parsers.insights_client_conf.InsightsClientConf` instead.

**Review by xiangce — CHANGES_REQUESTED** [review]

**Review by xiangce — APPROVED** [review]

**Pull request merged**